docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+# Beginners Contributing Guide
+
+## Prerequisites:
+* Docker is already running in your environment
+* You have a [Github](https://github.com) account and are logged in
+* This guide was written for linux
+
+__Note__: **This document will use variables thought out make sure to change them as needed.**
+
+### 1. Install Git
+This section will install Git.
+
+0. Log in to the server you are going to use for Git.
+
+0. Type `add-apt-repository ppa:git-core/ppa`.
+
+0. Type `apt-get install git` git will be installed.
+
+0. Type `git config --global user.email "email@example.com"` to set an email address that will be used when you post to GitHub.
+
+0. Type `git config --global user.email` to verify the email address.
+
+#### 1.1 Fork & Clone the Self-Hosted Show wiki
+This section will Fork the entire Self-Hosted Show wiki to your Github account, and then clone the fork in order to enable local editing.
+
+0. Navigate to the [upstream](https://github.com/selfhostedshow/wiki).
+
+0. Click ![Fork](images/fork.PNG) in the top right to copy the "wiki" repository to your Github account.
+
+0. Log in to the server you are going to use for Git and `cd` to the directory you want to download the wiki repository to.
+
+0. Type `git clone https://github.com/YOURGITHUBSUSERNAME/wiki && cd wiki`.
+
+0. Verify the "dev" branch is checked out type `git checkout -b dev`.
```

Comment:
This one should be `git checkout dev`, because a `dev` branch already exists.

Reply:
done